File: app/src/main/java/com/trevorwiebe/trackacow/utils/LotViewHolder.java
package com.trevorwiebe.trackacow.utils;

import androidx.recyclerview.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.trevorwiebe.trackacow.R;


public class LotViewHolder extends RecyclerView.ViewHolder {

    public TextView lotName;
    public ImageView reorder;
    public View lotMoveView;

    public LotViewHolder(View view) {
        super(view);

        lotName = view.findViewById(R.id.item_lot_name);
        reorder = view.findViewById(R.id.lot_reorder);
        lotMoveView = view.findViewById(R.id.lot_move_view);
    }
}

File: app/src/main/java/com/trevorwiebe/trackacow/presentation/fragment_move/utils/LotViewHolder.kt
package com.trevorwiebe.trackacow.presentation.fragment_move.utils

import android.view.View
import android.widget.ImageView
import androidx.recyclerview.widget.RecyclerView
import android.widget.TextView
import com.trevorwiebe.trackacow.R
import com.trevorwiebe.trackacow.domain.models.lot.LotModel

class LotViewHolder(view: View) : RecyclerView.ViewHolder(view) {
    var lotName: TextView
    var reorder: ImageView
    var lotMoveViewBottom: View
    lateinit var lotModel: LotModel

    init {
        lotName = view.findViewById(R.id.item_lot_name)
        reorder = view.findViewById(R.id.lot_reorder)
        lotMoveViewBottom = view.findViewById(R.id.lot_move_view)
    }
}